xml/System/MathF.xml

```diff
@@ -989,10 +1400,30 @@ The value of this field is 3.14159274.
         <Parameter Name="x" Type="System.Single" />
       </Parameters>
       <Docs>
-        <param name="x">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="x">A single-precision floating-point number to be rounded.</param>
+        <summary>Rounds a single-precision floating-point value to the nearest integral value, and rounds midpoint values to the nearest even number.</summary>
+        <returns>The integer nearest <paramref name="x" />. If the fractional component of <paramref name="x" /> is halfway between two integers, one of which is even and the other odd, then the even number is returned. Note that this method returns a <see cref="T:System.Single" /> instead of an integral type.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+This method uses the default rounding convention of <xref:System.MidpointRounding.ToEven?displayProperty=nameWithType>. See [Midpoint values and rounding conventions](#midpoint-values-and-rounding-conventions) for information about rounding numbers with midpoint values.
+
+[!INCLUDE[midpoint-rounding](~/includes/floating-point-equality.md)]
+
+If the value of the `x` argument is <xref:System.Single.NaN?displayProperty=nameWithType>, the method returns <xref:System.Single.NaN?displayProperty=nameWithType>. If `x` is <xref:System.Single.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Single.NegativeInfinity?displayProperty=nameWithType>, the method returns <xref:System.Single.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Single.NegativeInfinity?displayProperty=nameWithType>, respectively.  
+  
+ ]]></format>
+        </remarks>
+        <block subset="none" type="usage">
+          <para>Because of the loss of precision that can result from representing decimal values as floating-point numbers or performing arithmetic operations on floating-point values, in some cases the <see cref="M:System.MathF.Round(System.Single)" /> method may not appear to round midpoint values to the nearest even integer. In the following example, because the floating-point value .1 has no finite binary representation, the first call to the <see cref="M:System.MathF.Round(System.Single)" /> method with a value of 11.5 returns 11 instead of 12.  
```

Comment:
```suggestion
          <para>Because of the loss of precision that can result from representing decimal values as floating-point numbers or performing arithmetic operations on floating-point values, in some cases the <see cref="M:System.MathF.Round(System.Single)" /> method may not appear to round midpoint values to the nearest even integer.  
```
When converted to a Single, the example doesn't lose precision.